frontend/src/components/browse/BrowseContent.tsx

```diff
@@ -69,7 +69,6 @@ export default function BrowseContent({
   initialOrganizations,
   initialProjects,
   customSearchBarLabels,
-  errorMessage,
```

Comment:
`errorMessage` is provided by the FilterContext. It does not need to be propagated